### ISSUE TITLE
Make Test Driven Development More Prominent in Exercise Instructions

### ISSUE THREAD

**Issue by Stargator** [exercism/dart#376]
Recently, we've had a few students confused that all but the first test is skipped. They were not aware of Exercism's approach to nudge them towards TDD. (See #375 #347)

For this track, I think we can add the instructions **and** reasoning for this approach more prominent. Correctly, the student would have to look fairly deeply into the track page to find any reference.

To find instructions, they would have to go to the main Dart Track page and look for a link at the bottom on the right-hand side. Click that link. Then from there, you can click on [Testing on the Dart Track](https://exercism.org/docs/tracks/dart/tests).

That's a pretty well obfuscated for new students to any track page.

What this track can do, would be to add a blurb about the reasoning **and** track-specific guidance on how to practice TDD with the API used in the `test` package.

Any other thoughts or ideas related to this from other @exercism/dart ?

@ErikSchierboom, @iHiD, @angelikatyborska have you seen other students for other tracks not understanding why all tests after the first were skipped?

@synaptiko had a suggestion for the exercism tool:
> maybe `exercism submit` command should report it to you (it should check that you don't have any skipped tests before the submit).

**Comment by synaptiko** [comment]
My two cents as a fresh student on Exercism tool. When I started the track, I saw something like this:
![image](https://user-images.githubusercontent.com/1593280/149636486-a6ea616d-5103-49b7-a703-2d2bbb95dc32.png)

But the page where the skipped tests are described is hidden under a very generic link. I somehow thought it's a link to the official Dart documentation which I already had opened.

For each exercise I was reading both `README.md` and `HELP.md` files, that's where I would expect this kind of instructions.

And as Stargator mentioned above, I think `exercism submit` would be great place for some report of how many tests are passing/skipped/failing etc. and that in order to complete the exercise you are supposed to have all of them passing.

**Comment by iHiD** [comment]
(cc @taiyab - maybe we add links to all the docs here rather than just to the docs section?)

**Comment by ErikSchierboom** [comment]
> @ErikSchierboom, @iHiD, @angelikatyborska have you seen other students for other tracks not understanding why all tests after the first were skipped?

Yeah, this happens from time to time. Tracks have different "solutions" to this, including adding information to the shared tests (see [this example](https://github.com/exercism/csharp/blob/main/exercises/shared/.docs/tests.md)), and other tracks have some comments added to their implementation or test files.
